psydac/fem/tensor.py

```diff
@@ -702,6 +710,16 @@ def integral(self, f, *, nquads=None):
     #--------------------------------------------------------------------------
     # Other properties and methods
     #--------------------------------------------------------------------------
+
+    @property
+    def is_multipatch(self):
+        return False
+
+    @property
+    def is_vector_valued(self):
+        return False
```

Comment:
These two properties should be moved up a bit, to the section "Abstract interface: read-only attributes".

Reply:
Done in 55fd2c9